.gitignore

```diff
@@ -30,3 +30,7 @@
 /coverage
 
 /node_modules/*
+
+
+# Ignore IDE files
+/.idea/*
```

Comment:
You should use gitignore_global for non app files. @issyl0 did work making this consistent across gov.uk a few weeks ago: https://github.com/search?l=&q=gitignore_global+user%3Aalphagov&type=Commits

Reply:
Thanks Kevin, I wasn't aware of .gitignore_global. I'll drop this commit.